core/src/test/resources/Migrator/AddTableForDatabaseContext.xml

```diff
@@ -0,0 +1,22 @@
+<databaseChangeLog
+	xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd"
+	objectQuotingStrategy="QUOTE_ONLY_RESERVED_WORDS">
+
+	<changeSet id="IAF_Util:1:${tableName}" author="Erik van Dongen" dbms="mssql">
```

Comment:
I would not put `IAF_Util` in the changeset-ids, these changes are for unit testing and not for IAF-Test. I would suggest making it specific to this file.

Reply:
Goede tips, dank je!